src/daemon/filter_chain.c

```diff
@@ -219,7 +219,7 @@ static int fc_config_add_match(fc_match_t **matches_head, /* {{{ */
   }
 
   if (ptr == NULL && IS_TRUE(global_option_get("AutoLoadPlugin"))) {
-    char plugin_name[NAME_MAX];
+    char plugin_name[DATA_MAX_NAME_LEN];
```

Comment:
Something went wrong, rebase probably shouldn't touch this file.

Reply:
Damn, this should have gone into a different branch. I'll clean this up.